SignalServiceKit/src/Storage/TSRecipient.h

```diff
@@ -1,12 +1,14 @@
 //
-//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
 //
 
+#import "TSYapDatabaseObject.h"
+
 NS_ASSUME_NONNULL_BEGIN
 
 // Some lingering TSRecipient records in the wild causing crashes.
 // This is a stop gap until a proper cleanup happens.
-@interface TSRecipient : NSObject <NSCoding>
+@interface TSRecipient : TSYapDatabaseObject
```

Comment:
Just a reminder that this is going away.

Reply:
Yup!